v19.2/build-a-java-app-with-cockroachdb.md

```diff
@@ -258,6 +258,21 @@ try (PreparedStatement pstmt = connection.prepareStatement("INSERT INTO accounts
 }
 ~~~
 
+### Retrieve large data sets in chunks using cursors
+
+<span class="version-tag">New in v19.2:</span> CockroachDB now supports the Postgres wire-protocol cursors for implicit transactions and explicit transactions executed to completion. This means the [PGJDBC driver](https://jdbc.postgresql.org) can use this protocol to stream queries with large result sets. This is much faster than [paginating through results in SQL using `LIMIT .. OFFSET`](selection-queries.html#paginate-through-limited-results).
+
+For instructions showing how to use cursors in your Java code, see [Getting results based on a cursor](https://jdbc.postgresql.org/documentation/head/query.html#query-with-cursor) from the PGJDBC documentation.
+
+Note that interleaved execution (partial execution of multiple statements within the same connection and transaction) is not supported when [`Statement.setFetchSize()`](https://docs.oracle.com/javase/8/docs/api/java/sql/Statement.html#setFetchSize-int-) is used. For example, the following will not work:
```

Comment:
I would not include the code example until we have one that is known to not work. This is because I think it is more misleading than helpful. It may cause people to think they can't execute multiple statements in the same txn.

Reply:
That makes sense.  I've removed it.